sym_ewram.txt

```diff
@@ -1130,7 +1130,7 @@ gUnknown_203AB40: @ 203AB40
 gUnknown_203AB44: @ 203AB44
 	.space 0x4
 
-gUnknown_203AB48: @ 203AB48
+gDiploma: @ 203AB48
```

Comment:
This is actually static. You may want to delete it here and handle it in src/diploma.c, see this line https://github.com/pret/pokefirered/blob/master/src/teachy_tv.c#L70